testing/zerotier-one/zerotier-one.initd

```diff
@@ -1,11 +1,14 @@
 #!/sbin/openrc-run
 
-depend() {
-    provide net
-    use network
-    before firewall netmount
-}
-
+name=zerotier-one
 command="/usr/sbin/zerotier-one"
+command_args="$zerotier_opts"
```

Comment:
No, capitalized variables are just legacy practice. However, this variable is unnecessary, it’d be better to just remove this line and let users define `command_args` in conf.d if needed.